DXMainClient/DXGUI/Multiplayer/GameListBox.cs

```diff
@@ -332,6 +359,19 @@ public override void Draw(GameTime gameTime)
                         height, txPasswordedGame.Width, txPasswordedGame.Height),
                         Color.White);
                 }
+                else
+                {
+                    Logger.Log("GameDifficulty **  Drawing game difficulty icon: " + hostedGame.GameDifficulty);
```

Comment:
DO NOT

```suggestion
```

Reply:
If you need an explanation -- you're logging every single frame, this will absolutely tank the performance.